spec/timezone.html

```diff
@@ -16,6 +16,8 @@ <h1>Time Zone Names</h1>
     </p>
     <p>
       Built-in time zones may be <dfn variants="named time zone">named time zones</dfn>, represented by Strings for which IsAvailableTimeZoneName returns *true*.
+      Caller-supplied names of named time zones can be aliases for <dfn variants="canonical time zone">canonical time zones</dfn>.
+      CanonicalizeTimeZoneName is used to translate aliases into canonical time zone names.
       They may also be <dfn variants="offset time zone">offset time zones</dfn>, represented by Strings for which IsTimeZoneOffsetString returns *true*.
```

Comment:
Interesting. I had been thinking of aliases as only the TZDB Links, not variations in case. What's the advantage in making case variations also be called "aliases"?

Reply:
I don't care about the specific term, just that the text is unambiguous and supports all relevant constraints. But whatever the term is, how will you define it if not by the output of CanonicalizeTimeZoneName? Is there a relevant difference between canonicalization of "utc" to "UTC" vs. "Etc/UTC" to "UTC"?

Reply:
Maybe something like this text below?  (Note: may be changed depending on #2509 outcome.)

Some of those Strings are <dfn variants="canonical time zone name">canonical time zone names</dfn> for which CanonicalizeTimeZoneName returns its input unmodified, while each other one is an <em>alias</em> for a canonical time zone name.
An alias may be an ASCII-case-insensitive match for the canonical time zone name, such as *"europe/london"* for the canonical time zone name *"Europe/London"*.
An alias may also be a "Link" in the IANA Time Zone Database, such as *"Asia/Ulan_Bator"*  for the canonical time zone name *"Asia/Ulaanbaatar"*.
For backwards compatibility reasons, *"UTC"* in ECMAScript is a canonical time zone name, even through *"Etc/UTC"* is canonical in the IANA Time Zone Database.